www/views/info.html

```diff
@@ -1,3 +1,45 @@
-<div id=info>
-<h1>Hello Fatimah!</h1>
-</div>
+<div class="main" id="info">
+      
+            <div class="container">
+                <h1 class="myh1"> Restaurant Information</h1>
+            </div>
+            <br>
+    
+            <div class="container">
+                <h2 class="myh2">Hours</h2>
+                <p>Mon-Thu&nbsp;<a class="note">7:30Am-8:00Pm.</a></p>
+                <p> Fri-sun &nbsp;<a class="note">8:00Am-9:00.</a></p>
+                <p>**<a class="red">Open Now</a>**</p>
```

Comment:
use <a> tags for html links only. Here you could use a <p> tag instead. Make sure it still has the class="red". do this for each <a> tag that doesn't have a link.